sspl_test/rabbitmq/consume.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3.6
```

Comment:
this abstraction is actually good, using kafka utility directly would need rework later for this consumer, if message broker changes to something else in future.